src/action/mod.rs

```diff
@@ -24,6 +24,7 @@ pub mod interfaces;
 pub mod metadata;
 pub mod startup;
 pub mod network;
+pub mod install_date;
```

Comment:
Any ideas how to make this one word? :D I was trying to come up with something but failed :(

Reply:
I also have no ideas whether it's possible to use a one-word name. Does it really matter? :)

Reply:
How about `insttime`? Unfortunately, I cannot say why it is better. I am just completely unable to justify this (which is generally not a good thing and making others follow this makes it even more terrible), but having single-word names for action modules just seems *right* and non-single-word names does not seem right.

I did some GitHub search and there is a non-zero amount of `insttime`s in peoples code, so its not completely unheard of. And it would nicely correspond to things like `uptime` or `boottime`.